pkg/auth/rhsso_authz_handler.go

```diff
@@ -65,6 +51,30 @@ func handleOwnershipQueryError(err error) (bool, error) {
 	return true, nil
 }
 
+func (a *AuthzHandler) OwnedBy(ctx context.Context, db *gorm.DB) *gorm.DB {
+	if identity.IsAdmin(ctx) {
+		return db
+	}
+	if a.isTenancyEnabled() {
+		return db.Where("org_id = ?", ocm.OrgIDFromContext(ctx))
+	} else {
```

Comment:
no need for the 'else'

Reply:
It was actually written like that on purpose so it is clear that the user filtering is not a common code